### DISCUSSION TITLE
錯誤

### DISCUSSION THREAD

**Discussion by Funwh** [hacksider/Deep-Live-Cam#612] [General]
![擷取](https://github.com/user-attachments/assets/b8560542-e756-411b-8021-f91658502a4b)

**Comment by hacksider** [comment]
you have to redownload

**Comment by Funwh** [comment]
但是我重新下載還是一樣

Kenneth Estanislao ***@***.***> 於 2024年9月20日 週五 22:10 寫道：

**Comment by Funwh** [comment]
是說已經解決下載問題了嗎

方文豪 ***@***.***> 於 2024年9月20日 週五 22:13 寫道：

**Comment by hacksider** [comment]
Sorry but I can't understand what you are saying

**Comment by Funwh** [comment]
I have re-downloaded it and it still doesn’t work.

Kenneth Estanislao ***@***.***> 於 2024年9月20日 週五 22:34 寫道：

**Comment by hacksider** [comment]
You have to download everything

**Comment by Funwh** [comment]
01、02、03、and Deep?

Kenneth Estanislao ***@***.***> 於 2024年9月20日 週五 22:52 寫道：

**Comment by Funwh** [comment]
Or are there other programs?

方文豪 ***@***.***> 於 2024年9月20日 週五 23:01 寫道：

**Comment by hacksider** [comment]
If you just want a single file download, I suggest getting the one on gumroad. Those packages are for Nvidia only though as of this moment.

**Comment by Funwh** [comment]
我是Nvidia只是不懂為何下載下來會出錯

Kenneth Estanislao ***@***.***> 於 2024年9月20日 週五 23:15 寫道：

**Comment by Funwh** [comment]
I am Nvidia but I don’t understand why there is an error when downloading

方文豪 ***@***.***> 於 2024年9月20日 週五 23:34 寫道：

**Comment by hacksider** [comment]
I suggest just build it from the github or just get the gumroad file ☺️

**Comment by Funwh** [comment]
how do i find the program

Kenneth Estanislao ***@***.***> 於 2024年9月20日 週五 23:37 寫道：

**Comment by Funwh** [comment]
How do I find Deep-Live-Cam on gumroad


方文豪 ***@***.***> 於 2024年9月21日 週六 上午6:05寫道：

**Comment by hacksider** [comment]
https://hacksider.gumroad.com/l/vccdmm

please be informed that this is a packaged version of 1.4 which is just the same as what is here on github without the installation of other stuffs. You just need to install CUDA (link is posted on the installation as well on gumroad)

**Comment by Funwh** [comment]
May I ask where the image to video file is located?

Kenneth Estanislao ***@***.***> 於 2024年9月22日 週日 下午12:18寫道：

**Comment by Funwh** [comment]
Is it not possible to change pictures to videos?

方文豪 ***@***.***> 於 2024年9月22日 週日 下午12:47寫道：

**Reply by hacksider** [reply]
Please check the readme for the features. This software didn't claim to have image to video features ☺️

**Comment by Funwh** [comment]
@@

Kenneth Estanislao ***@***.***> 於 2024年9月22日 週日 13:05 寫道：
